### PR TITLE
Add capabilities to report the integrals of FVFluxBCs. 

### LINKED ISSUE #28767: Add postprocessor that can integrate an FVFluxBoundaryCondition.
### Motivation

We have a lot of different boundary conditions in FV and many of them provide a flux on the boundary (radiative, convective, advective, diffusive). Would be nice to have a postprocessor that can just report the integrated value of these fluxes.

### Design

Create a postprocessor that reports these values.

### Impact

Will fix the issue coming from extrapolated values when using `VolumetricFlowRate` for reporting values on flux boundary conditions.

### PR DESCRIPTION
**Pull request by grmnptr**
Closes #28767

### CODE REVIEW

**Review by lindsayad — COMMENTED** [review]
very nice addition

File: framework/src/postprocessors/SideFVFluxBCIntegral.C [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    std::vector<FVFluxBC *> flux_bcs;
+    base_query.condition<AttribName>(name).queryInto(flux_bcs);
+    if (flux_bcs.size() == 0)
+      paramError("fvbcs",
+                 "The given FVFluxBC with name '",
+                 name,
+                 "' was not found! This can be due to the boundary condition not existing in the "
+                 "'FVBCs' block or the boundary condition not inheriting from FVFluxBC.");
+
+    _bc_objects.push_back(flux_bcs[0]);
```
[the remark this answers is not in the document]
Comment (grmnptr) [reply framework/src/postprocessors/SideFVFluxBCIntegral.C:59]: I think we will get an error if there are multiple boundary conditions with the same name.

File: test/tests/postprocessors/fvfluxbc_integral/fvfluxbc_integral.i [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+[Executioner]
+  type = Steady
+  solve_type = PJFNK
+  petsc_options_iname = '-pc_type -pc_hypre_type'
+  petsc_options_value = 'hypre boomeramg'
+  nl_abs_tol = 1e-14
+  nl_rel_tol = 1e-14
+  l_abs_tol = 1e-14
+  l_tol = 1e-6
```
[the remark this answers is not in the document]
Comment (grmnptr) [reply test/tests/postprocessors/fvfluxbc_integral/fvfluxbc_integral.i:53]: Not really, I'll loosen them a little.

File: framework/src/postprocessors/SideFVFluxBCIntegral.C (hunk 1 above)
[the remark this answers is not in the document]
Comment (grmnptr) [reply framework/src/postprocessors/SideFVFluxBCIntegral.C:59]: I added a comment for this.

**Review by lindsayad — APPROVED** [review]

**Comment by grmnptr** [comment]
Thanks for the review!

**Comment by loganharbour** [comment]
This broke parallel sweep tests. See https://civet.inl.gov/event/201868/
